packages/itwin/grouping-mapping-widget/src/widget/components/CustomCalculationTable.tsx

```diff
@@ -125,20 +123,30 @@ const CustomCalculationTable = ({
         ],
       },
     ],
-    [onCustomCalculationModify, setSelectedCustomCalculation],
+    [onClickModifyCustomCalculation],
   );
 
   return (
     <>
-      <Button
-        startIcon={<SvgAdd />}
-        styleType='high-visibility'
-        onClick={() => {
-          setGroupModifyView(PropertyMenuView.ADD_CUSTOM_CALCULATION);
-        }}
-      >
-        Add Custom Calculation
-      </Button>
+      <div className="gmw-custom-calc-table-toolbar">
```

Comment:
These toolbars should be a separate component. That should reduce code duplication between different property types.

Reply:
Separated it to its own component.